### PR TITLE
Bump emoji version to `14.0.0`

### PR DESCRIPTION
**Pull request by miyanokomiya**
- Remove `package-lock.json` that seems obsolete
- New emoji has 61x61 splitesheets

### CODE REVIEW

File: src/emoji-data-ts.ts [hunk 1]
```diff
@@ -43,8 +43,9 @@ export const categoriesData = [
   'Flags'
 ]
 
-export const sheetColumns = 60
-export const sheetRows = 60
+export const sheetColumns = 61
+export const sheetRows = 61
+console.log("sheetColumns", sheetColumns);
```
Comment (SatoshiKawabata) [inline src/emoji-data-ts.ts]: @miyanokomiya Could you remove `console.log`?

**Review by SatoshiKawabata — CHANGES_REQUESTED** [review]
I added a comment. Please check it. @miyanokomiya

File: src/emoji-data-ts.ts (hunk 1 above)
Comment (miyanokomiya) [reply src/emoji-data-ts.ts]: @SatoshiKawabata I've fixed & rebased. Thanks!

**Review by SatoshiKawabata — APPROVED** [review]
@miyanokomiya LGTM!

**Pull request merged**